src/modules/relationships.ts

```diff
@@ -29,36 +29,36 @@ export interface Relationship {
 }
 
 interface FetchRelationshipsAction extends Action<typeof GET_RELATIONSHIPS> {
-  payload: { items: Relationship[] };
+  payload: { items: Relationship[]; assetId: string };
 }
 
 type RelationshipActions = FetchRelationshipsAction;
 
 // Functions
-export function fetchRelationships() {
-  return async (dispatch: ThunkDispatch<any, void, AnyAction>) => {
-    const { project } = sdk;
-    try {
-      const response = await sdk.get(
-        `/api/playground/projects/${project}/relationships`
-      );
+// export function fetchRelationships() {
```

Comment:
Let's remove the commented out code?

Reply:
It will be added to be something clever soon ;)